featurebyte/query_graph/sql/adapter/databricks.py

```diff
@@ -172,7 +172,7 @@ def get_value_from_dictionary(
     @classmethod
     def convert_to_utc_timestamp(cls, timestamp_expr: Expression) -> Expression:
         # timestamps do not have timezone information
-        return timestamp_expr
+        return expressions.Cast(this=timestamp_expr, to=expressions.DataType.build("TIMESTAMP"))
```

Comment:
This is needed for the hive UDFs to work with the new `TIMESTAMP_NTZ ` type.
The behavior is consistent with using regular timestamps (`TIMESTAMP_LTZ`) since all spark sessions are always set to use the UTC timezone